### PR TITLE
Fix version parsing in build script

### PR DESCRIPTION
**Pull request by Xenorith**
after rearranging the pom.xml in sorted order, version parsing in build script returns the wrong value for version

fix by parsing the single version tag that is directly under the root project xml node

### CODE REVIEW

File: dev/scripts/src/alluxio.org/build/cmd/version.go [hunk 1]
```diff
@@ -32,17 +33,20 @@ func VersionF() error {
 	return nil
 }
 
-var versionRe = regexp.MustCompile(".*<version>(.*)</version>.*")
+// only one <version> tag is expected in the root pom.xml that is at the level directly under the root project node
+var versionRe = regexp.MustCompile(`^  <version>(.*)</version>$`)
```
Comment (dbw9580) [inline dev/scripts/src/alluxio.org/build/cmd/version.go:37]: So this requires the line starts with exactly two spaces?

**Review by dbw9580 — APPROVED** [review]
LGTM but I think using a xml parser would be more robust

File: dev/scripts/src/alluxio.org/build/cmd/version.go (hunk 1 above)
Comment (Xenorith) [reply dev/scripts/src/alluxio.org/build/cmd/version.go:37]: yup exactly two spaces and there's only one of them in the file

**Comment by Xenorith** [comment]
alluxio-bot, merge this please

**Pull request merged**